### PR TITLE
rewards landing

### PR DESCRIPTION
**Pull request by stormcloud266**
(Just need to update the image url with the assets repo file)

Screenshot 2024-10-03 at 6 46 33 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
* **#730** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/730?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/rewards/landing/screen.tsx
Comment (youngkidwarrior) [inline packages/app/features/account/rewards/landing/screen.tsx:1]: Let's move this to `packages/app/features/account/rewards/index.tsx`

then have 
`packages/app/features/account/rewards/activity/screen.tsx` 
and
``packages/app/features/account/rewards/lock-and-earn/screen.tsx`

File: packages/app/features/account/rewards/landing/screen.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+}: {
+  title: string
+  href: string
+  reward: string
+}) => {
+  return (
+    <YStack f={1} pos={'relative'} overflow="hidden" borderRadius={'$6'} backgroundColor={'$black'}>
+      <ImageBackground
+        source={{
+          uri: 'https://s3-alpha-sig.figma.com/img/a668/3317/565155fe10db1fac429e21394136ac1e?Expires=1728864000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=lHAA3T2b8PvLZWTVGP0egRR8mq~tvCuqZu5OLOCYVg9bf6IGUdnGHNU5RLf3zrLFe39BQVHKLWuIqGiVcObSgIbB867T58RDosDsW2OSZHFW2aCwqSfC6~AgoircuJoFSzCApoNYLMDxv4NCU3IBcgG3uyGjd245j0vAvKEaTAn0vZ4LGbzSxaq5--5jSMNurFDK~fxHN6qQYzPP-CcqAqKrylgOmbb2hLq8RCZoT8Aw37pS~RpASfa-TQ4yIBZLxZgjnu7Ty8xrQPl2iERY8l~zGaKEpcscj8tvx22NqytwsNEjqrlPSye87dEGz1gB~UZZGQGY235AW9kLun8ezA__',
```
Comment (youngkidwarrior) [inline packages/app/features/account/rewards/landing/screen.tsx:45]: Lets host this image in our assets repo

Comment (stormcloud266) [reply packages/app/features/account/rewards/landing/screen.tsx:45]: Was just waiting for that asset PR to get approved

File: packages/app/features/account/rewards/landing/screen.tsx
Comment (stormcloud266) [reply packages/app/features/account/rewards/landing/screen.tsx:1]: sounds good

**Review by youngkidwarrior — APPROVED** [review]

**Review by youngkidwarrior — APPROVED** [review]
This is looking good. Let's make the entire card clickable instead of just the arrow

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Oct 18, 5:36 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/730).

**Pull request merged**